draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,220 @@
+
+\begin{itemize}
+  \item At any time, Players can spend their Building Token to perform the same actions as if they visited a Trading Post.
+  \item When defending the town, and your Hero is not here, you can still use your Units (for free).
+  \item Each settlement is defend by clans Settlement defenders. (see below)
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+  \begin{itemize}
+   \item \textbf{\nth{1} Round:}
+    \begin{itemize}
+      \item At start of the Round, each player show up the \textit{nemesis map Tiles} (Far (II--III) Map Tile with Settlement of same Faction as players Towns). This Tile you can reveal freely.
+    \end{itemize}
+   \item \textbf{\nth{2} Round:}
+    \begin{itemize}
+      \item  First wave of enemy clan armies attacks the towns. (see below)
+      \item  Each player who defended his own town, during this Round can hire secondary Hero for 5 \svg{gold}, but you still have to use the population token.
+    \end{itemize}
+   \item \textbf{\nth{3} Round:}
+    \begin{itemize}
+      \item At start of the Round, each player shows up one Tile (Far (II--III) Map Tile. This Tile you can reveal freely. (Not matter if Tile is from pool or map template.)
+    \end{itemize}
+   \item \textbf{\nth{4} Round:}
+    \begin{itemize}
+      \item Second wave of enemy clan armies attacks the towns. (see below)
+      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
+     \end{itemize}
+   \item \textbf{\nth{6} Round:}
+    \begin{itemize}
+      \item Third wave of enemy clan armies attacks the towns. (see below)
+      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
+    \end{itemize}
+   \item \textbf{\nth{7} Round:}
+    \begin{itemize}
+      \item During this Round, Players can discard and draw Cards at the start of their turn twice rather than once.
+    \end{itemize}
+  \end{itemize}
+
+\subsection*{\MakeUppercase{Enemy's clans waves}}
+\begin{itemize}
+  \item If enemy's settlement is not conquered, then enemy clans armies are attacking your castle!
+  \item At the start of the each player's turn, the battle begins.
+  \item Before attack, you can use your building, spell book and population tokens, and use Might and Magic Cards, but you can't move with Hero.
+  \item Each player has to defend his own castle.
+  \item If you have citadel, \textit{Arrow tower} has decreased attack. The attack is for first/second/third wave as 2/3/4\svg{attack}.
+  \item The battle does not cost any extra movement points.
+  \item The strength of the siege army is determined by the table \textit{Enemy's Clans armies}.
+  \item If your main Hero is present in any of allied town (not settlement) and Clan wave attack your allied town where is no main Hero, you can participate on defending allied castle by using Might and Magic Cards (Simply: you defend your teammate's castle with their Units and your Hero's Cards).
```

Comment:
```suggestion
  \item If your main Hero is in an allied Town (not Settlement) when a Clan attacks another allied Town without a main Hero present, you may help defend using your Might and Magic Cards. You will use their Units and your Hero's Deck.
```

Reply:
Ah, i see bad wording now... 

It seems to be suposed,  that defending hero  control defending units... But there was supposed to   Defending player control their units and defending hero(player) can help them by using cards.

Reply:
In that case
```suggestion
  \item If your main Hero is in an allied Town (not Settlement) when a Clan attacks another allied Town without a main Hero present, you may help defend using your Might and Magic Cards. The allied Hero will use their Units, and you can help them by playing Cards from your hand during combat.
```
But what happens if that combat is lost? What should happen to your main Hero that was helping defend the allied Town?

Reply:
Oh, sorry, that would mean the scenario is lost. No need to add anything else.